src/markdown.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Compiler for Pandoc Markdown
+ */
+
+import stencila from '@stencila/schema'
+import * as pandoc from './pandoc'
+import { VFile } from './vfile'
+
+export const mediaTypes = []
+
+export const extNames = ['markdown']
```

Comment:
Yes, this is confusing. We have a Remark-based `md.ts` converter (that matches `.md` files implicitly due to the `mediaTypes` defined there) and a Pandoc-based `markdown.ts` converter (that  is currently not enabled because it is not included in `index.ts`.

There is a wider discussion, beyond the scope of this PR, about how/if we expose all the formats that Pandoc provides - and what to use when there is an overlap with other "native" converters like for Markdown. For now, I've remove this file.